attribute-lister.go

```diff
@@ -0,0 +1,7 @@
+package vugu
+
```

Comment:
How about adding:

```go
type VGAttributeListerFunc func() []VGAttribute
func (f VGAttributeListerFunc) ListAttributes() []VGAttribute { return f() }
```

This will allow things like `<div :='vugu.VGAttributeListerFunc(func() []VGAttribute { return ... } )' >`

(Same pattern as [http.HandlerFunc](https://golang.org/pkg/net/http/#HandlerFunc))